installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -22,7 +22,7 @@
 
         <util:FileSearch Variable="HasDotnet3122" Path="[ProgramFiles64Folder]dotnet\shared\Microsoft.WindowsDesktop.App\3.1.22\System.Xaml.dll" Result="exists" />
 
-        <Variable Name="InstallFolder" Type="string" Value="[LocalAppDataFolder]PowerToys"/>
+        <Variable Name="InstallFolder" Type="string" Value="[ProgramFiles6432Folder]PowerToys"/>
```

Comment:
(any particular reason why we chose 6432?)

Reply:
the old version is normal one (x64). imo a mistake

Reply:
Seemed like the best default, from https://wixtoolset.org/documentation/manual/v3/bundle/bundle_built_in_variables.html
> ProgramFiles6432Folder - gets the well-known folder for CSIDL_PROGRAM_FILES on 64-bit Windows and CSIDL_PROGRAM_FILESX86 on 32-bit Windows.

But we don't support 32, so no issues with making it 64.

Reply:
Oh, yeah, that seems great

Reply:
Pushed a commit to fix it.